src/components/AppFrame/useVersionInfo.js

```diff
@@ -0,0 +1,18 @@
+import { useSelector } from "react-redux";
```

Comment:
This hook should probably be renamed to something like `useApplicationHelpUrl` - since that's what it does. Always keep your names sensible, otherwise you will create problems for yourself and others down the line. And better a 200-character function name than one that doesn't tell you what it does!

Reply:
You're right... you know, I am also an experienced developer (mostly back-end though) and I fully agree that names should be appropriate. Still, naming things has always been a huge deal in my daily tasks :laughing:. Sometimes here in Montreal, we discuss about the right name for several minutes before to agree on something.

I admit that on this one, I did not pay much attention as I used basically the name as my actions, selectors & reducers, with that said, I like your suggestion and I will use it.